### DISCUSSION TITLE
Objects are not getting displayed in "GridView Preview"

### DISCUSSION THREAD

**Discussion by Matt-Sea** [pimcore/pimcore#15953] [Q&A / Support]
Hello all,

I have a beginner question regarding Pimcore Commuity Edition. Right now I am in the process of setting up my first data model. I am working as an admin.

When I click on a subfolder an then under "Data Objects", I see the table view of all created objects - this works fine.

However, if I now click on "Grid Options" and on "Preview Item", no objects are getting displayed. This leads to the fact that I cannot select an object for a "Many-To-One Relation", although it has been created.

My question therefore: How can I solve this problem?

Thanks!

Greetings,
Matt-Sea

**Comment by robertSt7** [comment]
Hello @Matt-Sea could you please check the browser console? Thanks

**Reply by Matt-Sea** [reply]
Hi robertSt7,

what I tried is the following: In the "Many-To-One Relation" I have linked to a class from the demo data (Manufacurer). The objects there are displayed in the preview. However, when I link to my newly created class, the objects are not displayed. So the error does not seem to be with the browser, but with my settings. But I copied the class settings exactly from the demo class Manufacturer.

I am running out of ideas what could be the reason of the problem....

Greetings,
Matt-Sea

**Comment by robertSt7** [comment]
Hi @Matt-Sea,

Sorry, I am not able to reproduce this. Could you please try it on our [public demo](https://demo.pimcore.fun/admin/login), record the steps and upload it here if you can reproduce it there?  
Thanks.

Greetings,
Robert

**Reply by Matt-Sea** [reply]
Hi Robert,

when I simulate the steps in the public demo, eveything works fine. I did the exact same steps on my local installation and Pimcore isn't showing any results.

Some pictures showing the behaviour...
1. I've created a new object which is stored and published
![image](https://github.com/pimcore/pimcore/assets/145099837/66fd282e-b9d0-4b80-9cbb-cc2420076228)

2. When I want to relate to that object in another objekt (product -> manufacturer) the object is not showing up
![image](https://github.com/pimcore/pimcore/assets/145099837/5e50d010-eda7-404d-9f5f-2227834e2eae)

Have you got any idea what I can adjust to make it work?

Thanks!

All the best,
Matt-Sea

**Comment by robertSt7** [comment]
Hi @Matt-Sea!

Is this the only `TestManufacturer` you have in your environment. May try to create a new Folder and add there some `TestManufacturer` Objects.

Best Greetings,
Robert

**Comment by Maksonis** [comment]
I think you need to run php bin/console pimcore:search-backend-reindex to re-index the backend search of picmore. You probably missed the cronjob which do that. Add additional messenger transport for backend search (pimcore_search_backend_message).
